spec/requests/api/v1/ecf_participants_spec.rb

```diff
@@ -322,15 +305,15 @@
           expect(ect_row["email"]).to eql ect.email
           expect(ect_row["full_name"]).to eql ect.full_name
           expect(ect_row["mentor_id"]).to eql mentor.id
-          expect(ect_row["school_urn"]).to eql mentor.participant_profiles[0].induction_records[0].school_cohort.school.urn
+          expect(ect_row["school_urn"]).to eql mentor.mentor_profile.current_induction_record.school_cohort.school.urn
           expect(ect_row["participant_type"]).to eql "ect"
-          expect(ect_row["cohort"]).to eql partnership.cohort.start_year.to_s
-          expect(ect_row["teacher_reference_number"]).to eql ect.teacher_profile.trn
-          expect(ect_row["teacher_reference_number_validated"]).to eql "false"
-          expect(mentor_row["eligible_for_funding"]).to be_empty
-          expect(mentor_row["pupil_premium_uplift"]).to eql "false"
-          expect(mentor_row["sparsity_uplift"]).to eql "false"
-          expect(mentor_row["training_status"]).to eql "active"
+          expect(ect_row["cohort"]).to eql mentor.mentor_profile.current_induction_record.schedule.cohort.start_year.to_s
+          expect(ect_row["teacher_reference_number"]).to be_empty
```

Comment:
I'm curious why this and the below are now expected to be empty across this spec and other specs?

Reply:
That was the `teacher_profile` factory that was setting a random TRN. Which I've removed because now it's passed in  in the service classes that creates the `teacher_profile` associated with the that participant.

Reply:
and the participants in this test aren't `:eligible_for_funding` which is the trait that would set that up.